apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -2036,10 +2036,13 @@ else:
         });
 
         $("#fileUploader").on('fb:updatePath', function (e, options) {
-          const uploadingToOzone = self.currentPath().startsWith("ofs://");          
+          const uploadingToOzone = self.currentPath().startsWith("ofs://");      
           if (uploadingToOzone && UPLOAD_CHUNK_SIZE) {
-            uploader.setOption('sizeLimit', UPLOAD_CHUNK_SIZE);
-          }  
+            uploader.setOption('sizeLimit', Math.min(UPLOAD_CHUNK_SIZE, UPLOAD_SIZE_LIMIT));
+          }
+          else if (UPLOAD_SIZE_LIMIT !== -1) {
```

Comment:
What would UPLOAD_SIZE_LIMIT=0 mean? I assume it would also prevent any uploading.

Reply:
Good point.

Reply:
User can't upload a file if the file size is zero. Hue already handle this case.

<img width="489" alt="Screenshot 2023-03-31 at 11 56 00 AM" src="https://user-images.githubusercontent.com/36241930/229040580-d64e55e6-9660-4b26-87f6-6efcfe77f914.png">